DEVELOPERS_GUIDE.md

```diff
@@ -0,0 +1,27 @@
+# Building for testing
+
+To test a branch in GitHub Actions, an updated `dist/index.js` file is required.
+
+```
+npm run build
+git add dist/.
+git commit -m "updating index.js"
+git log -q -n 1 dist/index.js | less -F
+```
+
+From the log output take note of the commit hash and push to GitHub
+
+In a test GitHub action you can use the branched build of the action by referencing the branch or commit hash, see [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses) for details on the `uses` syntax.
+
+```
+- name: Create Zip Package for Octopus Deploy
```

Comment:
worth updaing the sample to suit this repo?

Reply:
yep, copy paste noob!

Reply:
i did, but forgot to check it in